ui/admin/tests/acceptance/groups/create-test.js

```diff
@@ -52,11 +49,14 @@ module('Acceptance | groups | create', function (hooks) {
   });
 
   test('can create new group', async function (assert) {
-    const groupsCount = this.server.db.groups.length;
+    const groupsCount = this.server.schema.groups.all().models.length;
     await visit(urls.newGroup);
-    await fillIn('[name="name"]', 'group name');
-    await click('[type="submit"]');
-    assert.strictEqual(this.server.db.groups.length, groupsCount + 1);
+    await fillIn(commonSelectors.FIELD_NAME, commonSelectors.FIELD_NAME_VALUE);
+    await click(commonSelectors.SAVE_BTN);
+    assert.strictEqual(
+      this.server.schema.groups.all().models.length,
```

Comment:
We are repeating this within the test file. Could we define a let outside the test but within the describe context and tests reuse it?
[As we did here](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/host-catalogs/hosts/create-test.js#L72) and use it [here](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/host-catalogs/hosts/create-test.js#L77) and [here](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/host-catalogs/hosts/create-test.js#L87)

Reply:
ah yes good point, updated!